utils/templates/test_module+member.yml.in

```diff
@@ -0,0 +1,88 @@
+---
+- name: Test $name
+  hosts: ipaserver
+  become: true
+
+  tasks:
+
+  # CLEANUP TEST ITEMS
+
+  - name: Ensure $name NAME is absent
+    ipa$name:
+      name: NAME
+      state: absent
+
+  # CREATE TEST ITEMS
+
+  # TESTS
+
+  - name: Ensure $name NAME is present
+    ipa$name:
+      name: NAME
+      # Add needed parameters here
+    register: result
+    failed_when: not result.changed or result.failed
+
+  - name: Ensure $name NAME is present again
+    ipa$name:
+      name: NAME
+      # Add needed parameters here
+    register: result
+    failed_when: result.changed or result.failed
+
+  - name: Ensure $name NAME member PARAMETER2 VALUE is present
+    ipa$name:
+      name: NAME
+      PARAMETER2: VALUE
+      action: member
+    register: result
+    failed_when: not result.changed or result.failed
+
+  - name: Ensure $name NAME member PARAMETER2 VALUE is present again
+    ipa$name:
+      name: NAME
+      PARAMETER2: VALUE
+      action: member
+    register: result
+    failed_when: result.changed or result.failed
+
+  - name: Ensure $name NAME member PARAMETER2 VALUE is absent
+    ipa$name:
+      name: NAME
+      PARAMETER2: VALUE
+      action: member
+      state: absent
+    register: result
+    failed_when: not result.changed or result.failed
+
+  - name: Ensure $name NAME member PARAMETER2 VALUE is absent again
+    ipa$name:
+      name: NAME
+      PARAMETER2: VALUE
+      action: member
+      state: absent
+    register: result
+    failed_when: result.changed or result.failed
+
+  # more tests here
+
+  - name: Ensure $name NAME is absent
+    ipa$name:
+      name: NAME
+      state: absent
+    register: result
+    failed_when: not result.changed or result.failed
+
+  - name: Ensure $name NAME is absent again
+    ipa$name:
+      name: NAME
```

Comment:
The `standard password` should be added here: `ipaadmin_password: SomeADMINpassword`.